src/pages/docs/ai-transport/features/sessions-identity/identifying-users-and-agents.mdx

```diff
@@ -0,0 +1,449 @@
+---
+title: "Identifying users and agents"
+meta_description: "Establish trusted identity and roles in decoupled AI sessions"
+meta_keywords: "user authentication, agent identity, JWT authentication, token authentication, verified identity, capabilities, authorization, user claims, RBAC, role-based access control, API key authentication, message attribution"
+---
+
+To build secure AI applications, agents need to trust who sent each message and understand what that sender is authorized to do. Ably's identity system provides cryptographically-verified identities and custom user attributes through token-based authentication.
+
+This page shows you how to authenticate users and agents, assign verified identities, attach custom attributes, and access that identity information in your applications.
```

Comment:
We don't tend to have a "This page shows you" build up to a page in the docs, rather the first paragraph already establishes the problem and solution. The headings themselves also tell you what's coming or give you a clear indication of what sections you're going to read. 

I'd suggest removing the second paragraph (line 9) from this page. What do you think?